shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -526,7 +526,6 @@ - (void)setupShell:(std::unique_ptr<flutter::Shell>)shell
   _publisher.reset([[FlutterObservatoryPublisher alloc]
       initWithEnableObservatoryPublication:doesObservatoryPublication]);
   [self maybeSetupPlatformViewChannels];
-  _shell->GetIsGpuDisabledSyncSwitch()->SetSwitch(_isGpuDisabled ? true : false);
```

Comment:
why delete this line?

Reply:
Because that value is now getting set when the shell is created.